.github/workflows/gutenberg-packages-update.yml

```
@@ -0,0 +1,146 @@
+name: Gutenberg packages update
+
+on:
+  # Allow for the workflow to be manually run if needed.
+  workflow_dispatch:
+  schedule:
+    # Once a day (https://crontab.guru/once-a-day)
+    - cron: '0 0 * * *'
+
+jobs:
+  check-gutenberg-release:
+    name: Check for a new Gutenberg release
+    runs-on: ubuntu-latest
+    outputs:
+      latest-version: ${{ steps.latest-release.outputs.version }}
+      should-update: ${{ steps.release-status.outputs.outdated }}
+    steps:
+      - name: Get latest release version
+        id: latest-release
+        run: echo "::set-output name=version::$(gh api -X GET repos/wordpress/gutenberg/releases/latest --jq '.name')"
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+
+      - name: Get release version from last PR
+        id: last-release
+        run: |
+          PR_TITLE=$(gh api -X GET search/issues -f q='${{ env.QUERY }}' -f sort='created' -f order='desc' --jq '.items.[].title' | head -n 1)
+          LAST_VERSION=$(node -e "'$PR_TITLE' && console.log( '$PR_TITLE'.match('v(.+) release')[1] )")
```

What's the `'$PR_TITLE' &&` part for?

If the matched pull request contains any quotation marks (perhaps by malicious user), then this will cause an error.

Perhaps something like this would be better:

```suggestion
          LAST_VERSION=$(sed 's/.* \(v.*\).*/$1/' <<< "$PR_TITLE")
```

---

This could then also do a check to see if it the resulting `LAST_VERSION` matches `^v\d+(\.\d+)*$`

---

Done in https://github.com/ampproject/amp-wp/pull/6302/commits/f0a0dab229283346ef589aa3323d272c3061603a.